src/components/WorldListPage/WorldListPage.tsx

```diff
@@ -89,6 +97,10 @@ const WorldListPage: React.FC<Props> = props => {
     [deploymentsByWorlds, onUnpublishWorld]
   )
 
+  const handlePageChange = useCallback((_event: any, props: any) => {
```

Comment:
If you find it easier, you could use [the `pagination` hook here](https://github.com/decentraland/builder/blob/f04929ed72e867c11ca18a06e9bf2c0837e4da07/src/lib/pagination.ts).